CSharp/BatchExplorer/Helpers/BasicDataProvider.cs

```diff
@@ -1,137 +1,151 @@
-﻿//Copyright (c) Microsoft Corporation
```

Comment:
It seems like the CRLF is needed and the diff looks fine just showing changes to me if using something like VS built-in diff viewer.